uber/templates/regform.html

```diff
@@ -15,7 +15,6 @@
         var checked = $.field('staffing') && $.field('staffing').is(':visible:checked');
         setVisible('#departments', checked);
         if ($.field('no_cellphone')) {
-            setVisible($.field('no_cellphone').parents('.checkbox'), checked);
```

Comment:
This makes the no_cellphone checkbox appear if someone has marked themselves as volunteering (which, in the backend, is one thing we check when we require a cellphone number). We don't want to mess with this.